src/Umbraco.Web.UI/config/umbracoSettings.config

```diff
@@ -63,6 +63,12 @@
     <usernameIsEmail>true</usernameIsEmail>
     <!-- change in 4.8: Disabled users are now showed dimmed and last in the tree. If you prefer not to display them set this to true -->
     <hideDisabledUsersInBackoffice>false</hideDisabledUsersInBackoffice>
+    <userPasswordConfiguration
```

Comment:
We'll need to update this in the umbracoSettings.Release.config file too (it's hidden away underneath this file, you'll need to click the expand button beside it)

Reply:
Thanks, have updated.